### ISSUE TITLE
Support strict mode

### ISSUE THREAD

**Issue by tuukkamustonen** [purcell/airspeed#39]
Looks like [strict mode](http://velocity.apache.org/engine/1.7/user-guide.html#strict-reference-mode) is not supported, correct?

**Comment by tuukkamustonen** [comment]
I think https://github.com/purcell/airspeed/blob/master/airspeed/__init__.py#L607 would be the place to add it?

**Comment by purcell** [comment]
No, it's not supported, and yes, I think that would be the right place to make a change. It's not clear where we'd set and store a value indicating whether strict mode is enabled.
